tinygrad/runtime/ops_cuda.py

```diff
@@ -56,9 +56,10 @@ class PTXCompiler(Compiler):
   linearizer_opts = LinearizerOptions("CUDA", suffix="PTX", global_max=[65535, 65535, 2147483647], local_max=[64, 1024, 1024], shared_max=49152)
   def __init__(self, arch:str):
     self.arch = arch
+    self.version = "7.8" if arch >= "sm_89" else "7.5"
```

Comment:
Err, what do you think that >= does?

Reply:
This should be accurate for existing architectures ;) but sure, I'll adjust to actually parse the integer version

Reply:
Updated

Reply:
oh haha I thought there was a 10, I was confused with cuda versions.